go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rollkit/rollkit
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```

Comment:
We cannot make this upgrade yet since the current latest version of `ignite` does not support go 1.21. Please revert the changes in this file.

Reply:
@Manav-Aggarwal Done